#! /bin/sh /usr/share/dpatch/dpatch-run
## 23-pulse_stubs_fix.dpatch by Dinxter <dinxter@hotmail.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adapt to PulseAudio 0.9.16 API change.

@DPATCH@
diff -urNad virtualbox-ose-3.0.2-dfsg~/src/VBox/Devices/Audio/pulse_stubs.c virtualbox-ose-3.0.2-dfsg/src/VBox/Devices/Audio/pulse_stubs.c
--- virtualbox-ose-3.0.2-dfsg~/src/VBox/Devices/Audio/pulse_stubs.c	2009-08-06 14:27:01.000000000 +0100
+++ virtualbox-ose-3.0.2-dfsg/src/VBox/Devices/Audio/pulse_stubs.c	2009-08-06 14:27:26.000000000 +0100
@@ -43,7 +43,7 @@
 
 PROXY_STUB     (pa_stream_connect_playback, int,
                 (pa_stream *s, const char *dev, const pa_buffer_attr *attr,
-                 pa_stream_flags_t flags, pa_cvolume *volume, pa_stream *sync_stream),
+                 pa_stream_flags_t flags, const pa_cvolume *volume, pa_stream *sync_stream),
                 (s, dev, attr, flags, volume, sync_stream))
 PROXY_STUB     (pa_stream_connect_record, int,
                 (pa_stream *s, const char *dev, const pa_buffer_attr *attr,
